### ISSUE TITLE
usdview does not work correctly due to the old version USD 23.11

### ISSUE THREAD

**Issue by bazhutkin** [Autodesk/maya-usd#3978]
usdview does not work correctly due to changes in PySide6. Because of this, in the current version 0.30.0 the viewport does not display the model.

This problem has already been described and solved:
https://github.com/PixarAnimationStudios/OpenUSD/commit/abb175da3587d3e21111f0d0b753fb2dd965d7dc

I replaced stageView.py file from the current version of USD 24.11 in the folder C:\Program Files\Autodesk\MayaUSD\Maya2025\0.30.0\mayausd\USD\lib\python\pxr\Usdviewq and usdview worked fine.

Please build the plugin on the current version of Pixar USD.

Windows 11, Maya 2025.3, Maya USD 0.30.0, Pixar USD 23.11

**Comment by santosd** [comment]
Thank you @bazhutkin for reporting this issue. Because we use specific builds of USD  inside of Maya we usually only update USD per release of Maya so that we don't break API compatibility between one update of a specific version of Maya to the next. Maya 2025.3 is on USD version 23.11 therefore any new version of MayaUSD for Maya 2025.3 will continue to be built against USD 23.11. However, if users have the ability to build MayaUSD on their own they can build against any version they would like.

**Comment by santosd** [comment]
On another note, I was not able to reproduce the issue that you are experiencing. Are you using the following for launching USDView?
https://help.autodesk.com/view/MAYAUL/2025/ENU/?guid=GUID-C452B331-7C4D-4B58-A1BA-D03775F266A8

**Comment by bazhutkin** [comment]
Yes, I tested all variants, including this unmodified code from the documentation. And every time this bug was reproduced.
